asm-primary/cluster/istio-operator.yaml

```diff
@@ -20,24 +20,19 @@ spec:
   profile: asm-gcp # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.profile"}
   hub: gcr.io/gke-release/asm # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.hub"}
   tag: 1.6.2-asm.0 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.tag"}
+  components:
+    pilot:
+      k8s:
```

Comment:
Is this "k8s" needed?

Reply:
Yes, verified by comparing both outputs from this istio-operator.yaml with --set values.pilot.env.SPIFFE_BUNDLE_ENDPOINTS.